src/time/span.cr

```diff
@@ -84,70 +87,14 @@ struct Time::Span
     @nanoseconds = nanoseconds.to_i32
   end
 
-  def self.new(*, nanoseconds : Int)
-    new(
-      seconds: nanoseconds.to_i64.tdiv(NANOSECONDS_PER_SECOND),
-      nanoseconds: nanoseconds.to_i64.remainder(NANOSECONDS_PER_SECOND),
-    )
-  end
-
-  private def self.compute_seconds!(days, hours, minutes, seconds)
-    compute_seconds(days, hours, minutes, seconds, true).not_nil!
-  end
-
-  private def self.compute_seconds(days, hours, minutes, seconds, raise_exception)
-    # TODO once overflow is the default this can be refactored
```

Comment:
I decided to refactor this method upon seeing this TODO - I remembered that overflowing as a default had been added just recently